### ISSUE TITLE
Weird spacing in hinted Inter 4.0 beta 9g

### ISSUE THREAD

**Issue by AchillesBoi** [rsms/inter#585]
After updating to the latest 4.0 beta 9g I noticed that some of the words on my screen looked odd. It seems there is uneven spacing between some letters, specifically letters after **'m'** and **'r'** as shown in the screenshot (top one is latest beta, bottom is previous).

![example](https://github.com/rsms/inter/assets/55808665/70cf1ca4-882b-47e4-b000-0bb8222deb0f)
(GNOME 42.5)

Is this intended behaviour? I'm assuming the recent weight changes have something to do with the spacing of the letters when hinted.

**Comment by rsms** [comment]
Let's keep the conversation in one place: https://github.com/rsms/inter/discussions/463#discussioncomment-6154396

**Comment by rsms** [comment]
Closing for now as this is most likely not a "bug".
Happy to reopen if we have a concrete explanation of the issue and its cause.

**Comment by AchillesBoi** [comment]
Referencing #463, I request that this issue be reopened:

Letter spacing is different between Beta 1-8 and Beta 9 in both hinted and unhinted formats. We have yet to determine if this new spacing is intentional or merely a side-effect of different font weights used between betas.

Betas 1 though 8
![Screenshot from 2023-06-12 16-33-36](https://github.com/rsms/inter/assets/55808665/c5044be8-861d-4a85-9976-6d20cad326b1)

Beta 9
![Screenshot from 2023-06-12 16-32-52](https://github.com/rsms/inter/assets/55808665/90c93766-3622-4b13-842d-a0d64b762cb2)

There is also the issue of monospaced numbers being used in place of regular ones, inconsistently across different apps and UI elements. Here's the best example of this:

You can see monospaced "1"s in the clock and calendar but not in the file manager.
![Screenshot from 2023-06-12 16-06-09](https://github.com/rsms/inter/assets/55808665/982b019d-84b0-4d32-b156-14b57ce1687a)

How it looked like in Betas 1 through 8:
![Screenshot from 2023-06-12 13-10-03](https://github.com/rsms/inter/assets/55808665/1964fa42-8b1c-430d-8a4f-df53304c37bd)

**Comment by rsms** [comment]
Regarding spacing:

What you can do to help is to run [ttfautohint](http://freetype.org/ttfautohint/) with various settings to try and find "the right setting" that generates the desired results for you. Basically you'll want to do this:

```
ttfautohint --reference extras/ttf/Inter-Regular.ttf \
  extras/ttf/Inter-Regular.ttf my-Inter-Regular.ttf
```

Test `my-Inter-Regular.ttf`

Repeat.

Tune various parameters to adjust the autohinter. See http://freetype.org/ttfautohint/doc/ttfautohint.html

**Comment by rsms** [comment]
Regarding "one" with serif, this is not a bug. This is the intended design. The calendar UI you are using has `tnum` enabled; find a way to disable it.

**Comment by AchillesBoi** [comment]
> Regarding "one" with serif, this is not a bug. This is the intended design. The calendar UI you are using has tnum enabled; find a way to disable it.

Was `tnum` added to Inter only recently with Beta 9? This would explain why the other betas did not have `tnum` enabled in the clock before.

Regarding spacing, I also figured out that even the early Beta 9s did not have this spacing issue and I did some digging and found out that some changes were done very recently in the newest Beta 9s that changed the font weight to fix the presentation of the font (or something like that). So that explains the spacing; it's not a bug, it's just that the font just got a tiny little bit thicker.

Are you able to confirm these changes?
